### ISSUE TITLE
Add alt text for profile pictures in profile page for WCAG Accessibility

### ISSUE THREAD

**Issue by ozdentarikcan** [bounswe/bounswe2024group1#478]
# 📌 Alt text for profile pictures in profile page for WCAG Accessibility
## 🏡 Component 
- [ ] Backend
- [X] Frontend
- [ ] Mobile
- [ ] Design
- [ ] Documentation 
- [ ] Other

## 📚 Description
We have chosen "Accessibility" (WCAG) as the Web Standard, so we will add alt-texts to the images in the frontend pages.
This task is for the profile pictures in the Profile Page.

## 📋 Acceptance Criteria
- [ ] Alt texts exist for profile pictures in profile pages.
- [ ] Acceptable alt texts should have the user name information and specify that the page is the profile page.

## 📃 Related requirements
[Link to section](https://github.com/bounswe/bounswe2024group1/wiki/Requirements#34-accessibility)
- #### 3.4.1. The web system shall implement the WCAG 2.2 standard to ensure accessibility.

    - #### 3.4.2. This includes but is not limited to:

      - 3.4.2.1. The System shall provide aria labels for all icon buttons.
      - 3.4.2.3. The System shall adopt a reasonable hierarchy for screen reader navigation.
### 📅 Deadline
- **Estimated hours of work**: 1 hour
- **Implementation Deadline:** November 5
- **Review Deadline:** November 5  

## 👁‍🗨 Reviewer
@asligook

**Comment by ozdentarikcan** [comment]
Closed with commit https://github.com/bounswe/bounswe2024group1/pull/475/commits/fd75ad2edabf807883627fa96d67f50e89925137
